source/Cosmos.IL2CPU/ILOp.cs

```diff
@@ -707,14 +730,18 @@ public static uint SizeOfType(Type aType)
             //}
             if (aType.IsEnum)
             {
-                return SizeOfType(aType.GetField("value__").FieldType);
+                result = SizeOfType(aType.GetField("value__").FieldType);
+                goto End;
             }
             if (aType.IsValueType && aType != typeof(ValueType))
             {
-                // structs are stored in the stack, so stack size = storage size
-                return GetStorageSize(aType);
+                result = GetStorageSize(aType);
+                goto End;
             }
-            return 4;
+
+        End:
```

Comment:
Just asking, is it still a good practice to use goto and labels? Isn't it to ban?